pkg/tuple/tuple_test.go

```diff
@@ -468,3 +468,75 @@ func TestTypedPublicWildcard(t *testing.T) {
 	require.Equal(t, "group:*", TypedPublicWildcard("group"))
 	require.Equal(t, ":*", TypedPublicWildcard("")) // Does not panic
 }
+
+func TestParseTupleString(t *testing.T) {
+	tests := []struct {
+		name        string
+		str         string
+		tuple       *openfgav1.TupleKey
+		expectedErr bool
+	}{
+		{
```

Comment:
can you add one test for `user:*` please?

Reply:
Added this test https://github.com/openfga/openfga/pull/1586/files#diff-57a1b9934b1de403d56fdc98ba5b12686658b098e57e021f371a5dff321f0652R387-R391 in the PR targeting `main`.